lib/mpl_toolkits/axes_grid1/parasite_axes.py

```diff
@@ -327,9 +327,10 @@ def _remove_method(h):
         return ax2
 
     def get_tightbbox(self, renderer, call_axes_locator=True):
-        bbs = [ax.get_tightbbox(renderer, call_axes_locator)
+        bbs = [ax.get_tightbbox(renderer, call_axes_locator=call_axes_locator)
                for ax in self.parasites]
-        bbs.append(super().get_tightbbox(renderer, call_axes_locator))
+        bbs.append(super().get_tightbbox(renderer,
+                    call_axes_locator=call_axes_locator))
```

Comment:
similar issue here, the alignment of the keyword argument is wonky.

Reply:
Fixed